frontend/src/Settings/Profiles/Release/ReleaseProfile.js

```diff
@@ -79,6 +80,9 @@ class ReleaseProfile extends Component {
         overlayContent={true}
         onPress={this.onEditReleaseProfilePress}
       >
+        <div className={styles.name}>
```

Comment:
After playing with it for a moment we should conditionally render this so we don't have a margin without  a name

```
{
  name ?
    <div className={styles.name}>
      {name}
    </div> :
  null
}
```

Please also add a blank line between the closing brace and the next `<div>`.

Reply:
I've made the requested change. Note that on the `null` line, the linter was not satisfied with the indentation you chose, so I aligned it with the `</div>` and it was happy with that.

Let me know if there's anything else you'd like changed. Thank you for the quick turnaround.